packages/contractkit/src/identity/offchain-data-wrapper.ts

```diff
@@ -61,9 +62,9 @@ export default class OffchainDataWrapper {
       return Err(new NoStorageRootProvidedData())
     }
 
-    const item = (await Promise.all(storageRoots.map((_) => _.readAsResult(dataPath)))).find(
-      (_) => _.ok
-    )
+    const item = (
+      await Promise.all(storageRoots.map((_) => _.readAndVerifySignature(dataPath, buildTypedData)))
+    ).find((_) => _.ok)
```

Comment:
`_` usually means an unused variable, or the `lodash` library. I suggest using a single-letter variable here.
```suggestion
      await Promise.all(storageRoots.map((s) => s.readAndVerifySignature(dataPath, buildTypedData)))
    ).find((s) => s.ok)
```